src/app/home/home.component.html

```diff
@@ -9,17 +9,50 @@
   ></biggive-hero-image>
 
   <div class="b-container">
-    <biggive-basic-card
-      space-below='4'
-      icon-colour='red'
-      background-image-url='/assets/images/card-background-cc.jpg'
-      [mainTitle]="highlightCard1Title"
-      [subtitle]='highlightCard1Subtitle'
-      button-url='/christmas-challenge-2022'
-      button-label='Double my donation'
-      button-colour-scheme='clear-primary'
-    >
-    </biggive-basic-card>
+
+    <biggive-grid
+      column-count="3">
+      <biggive-basic-card
+        space-below='4'
+        icon-colour='brand-3'
+        background-image-url='/assets/images/card-background-gmf.jpg'
+        main-title='Save the Date for the Green Match Fund'
+        subtitle='Applications open 9 Jan 2023'
+        button-url='https://blog.thebiggive.org.uk/green-match-fund/'
+        button-label='Read more'
+        button-colour-scheme='clear-primary'
+        class="basic-card-padding"
+      >
+      </biggive-basic-card>
+
+      <biggive-basic-card
+        space-below='4'
+        icon-colour='brand-5'
+        background-image-url='/assets/images/colour-orange.png'
```

Comment:
I wonder why we actually need an image of a plain colour. But I tried passing the `background-colour` prop instead with this #f07d00 colour and it it didn't work.

Reply:
`background-colour` will only work as a CSS style. See the component docs here: https://components-staging.thebiggivetest.org.uk/?path=/docs/components-content--basic-card-component. This is fine as is.